### PR TITLE
Remove Windows shortcuts during uninstall.

### PR DESCRIPTION
**Pull request by mitchell-as**
<table>
<td>
  <a href="https://activestatef.atlassian.net/browse/DX-2993" title="DX-2993" target="_blank">BugDX-2993</a>  Uninstall does not delete shortcuts
  </td></table>
  <br />

### CODE REVIEW

File: internal/osutils/autostart/autostart_windows.go [hunk 1]
```diff
@@ -59,8 +59,8 @@ func isEnabled(_ string, opts Options) (bool, error) {
 	return fileutils.FileExists(shortcutFilename(opts.Name)), nil
 }
 
-func autostartPath(name string, _ Options) (string, error) {
-	return shortcutFilename(name), nil
+func autostartPath(_ string, opts Options) (string, error) {
+	return shortcutFilename(opts.Name), nil
 }
```
Comment (mitchell-as) [inline internal/osutils/autostart/autostart_windows.go:64]: This bug gave me hours of grief :cry:

Comment (mitchell-as) [inline internal/osutils/autostart/autostart_windows.go:64]: This bug gave me hours of grief :cry:

File: test/integration/uninstall_int_test.go [hunk 2]
```diff
@@ -139,6 +139,11 @@ func (suite *UninstallIntegrationTestSuite) testUninstall(all bool) {
 		}
 	}
 
+	if runtime.GOOS == "windows" {
+		shortcutDir := filepath.Join(ts.Dirs.HomeDir, "AppData", "Roaming", "Microsoft", "Windows", "Start Menu", "Programs", "ActiveState")
```
Comment (MDrakos) [inline test/integration/uninstall_int_test.go:143]: Are we verifying that this file exists after install? Otherwise this could be a false positive.

**Review by MDrakos — CHANGES_REQUESTED** [review]

**Comment by mitchell-as** [comment]
Fair enough. I knew it was getting deleted through manual testing, but it doesn't hurt to have it in a test either.

**Review by MDrakos — APPROVED** [review]

**Pull request merged**